src/test/java/org/broadinstitute/consent/http/service/InstitutionServiceTest.java

```diff
@@ -38,40 +39,126 @@ private void initService() {
     service = new InstitutionService(institutionDAO);
   }
 
+  private Institution initMockModel() {
+    Institution mockInstitution = new Institution();
+    mockInstitution.setName("Test Name");
+    return mockInstitution;
+  }
+
   @Test
-  public void testCreateInstitution() {
+  public void testCreateInstitutionSuccess() {
+    Institution mockInstitution = initMockModel();
+    when(institutionDAO.findInstitutionById(anyInt())).thenReturn(mockInstitution);
     initService();
-    //mock findByInstitution (not insertInstitution) because that is the call that service.createInstitution returns
-    when(institutionDAO.findInstitutionById(anyInt())).thenReturn(getInstitutions().get(0));
-    Institution institute = service.createInstitution(anyString(), anyString(), anyString(), anyInt(), eq(new Date()));
-    assertEquals(getInstitutions().get(0), institute);
+    Institution institution = service.createInstitution(mockInstitution, 1);
+    assertNotNull(institution);
   }
 
   @Test
-  public void testUpdateInstitutionById() {
+  public void testCreateInstitutionBlankName() {
+    Institution mockInstitution = initMockModel();
+    mockInstitution.setName("");
+    initService();
+    try{
+      service.createInstitution(mockInstitution, 1);
+      Assert.fail("Institution CREATE should not succeed with an empty name");
+    } catch(Exception e) {
+      assertEquals("Institution name cannot be null or empty", e.getMessage());
+    }
+  }
+
+  @Test
+  public void testCreateInstitutionNullName() {
+    Institution mockInstitution = initMockModel();
+    mockInstitution.setName(null);
     initService();
-    //doNothing is default for void methods, no need to mock InstitutionDAO.updateInstitutionById
     try {
-      service.updateInstitutionById(anyInt(), eq("New Name"), anyString(), anyString(), anyInt(), eq(new Date()));
+      service.createInstitution(mockInstitution, 1);
+      Assert.fail("Institution CREATE should not succeed with an empty name");
     } catch (Exception e) {
-      Assert.fail("Update should not fail");
+      assertEquals("Institution name cannot be null or empty", e.getMessage());
     }
   }
 
-  @Test(expected = Throwable.class)
-  public void testUpdateInstitutionByIdFail() {
+  //NOTE: would like to add a test for name duplication, but exception will differ depending on whether name column has the UNIQUE keyword applied
```

Comment:
See `changelog-consent-61.0.xml` ... we do not have a unique constraint on that column (yet). Feel free to add one or file a new ticket to add it as an update along with new tests.